src/Makefile.am

```diff
@@ -196,10 +196,11 @@ settings_DATA=libhdf5.settings
 # Remove the generated .c file if errors occur unless HDF5_Make_Ignore
 # is set to ignore the error.
 H5Tinit.c: H5detect$(EXEEXT)
-	LD_LIBRARY_PATH="$$LD_LIBRARY_PATH`echo $(LDFLAGS) |                  \
-		sed -e 's/-L/:/g' -e 's/ //g'`"                               \
-	$(RUNSERIAL) ./H5detect$(EXEEXT)  $@  ||                               \
-	    (test $$HDF5_Make_Ignore && echo "*** Error ignored") ||          \
+	@if $(AM_V_P); then set -x; else echo "  GEN      H5Tinit.c"; fi;       \
```

Comment:
The new line on both rules is this one, where we check AM_V_P and modify the output